src/WP_CLI/CommandWithUpgrade.php

```diff
@@ -16,6 +16,9 @@ abstract class CommandWithUpgrade extends \WP_CLI_Command {
 
 	protected $chained_command = false;
 
+	// Invalid version message.
+	protected $invalid_version_message = 'version higher than expected';
```

Comment:
This should be a protected constant instead.
```suggestion
	protected const INVALID_VERSION_MESSAGE = 'version higher than expected';
```

Reply:
Class constant visibility was introduced in PHP `7.1` so we can't keep it as protected, should I just update it to be a `const`?

Reply:
Ah, yes, you are right. Just use `const` without modifier then.